elasticsearch/eval.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import json
+import os
+import re
+from itertools import chain
+
+import click
```

Comment:
Nether I nor Pycharm can find these imports: 

```
import click
import requests
from toolz.dicttoolz import assoc

import util
```
Maybe you forgot to copy some files from another repo.
Maybe update a requirements.txt somehwere? Possibly move requirements.txt out of the harvester folder to the root, if the same interpreter is to be used for all